pkgs/development/ocaml-modules/mirage-unix/default.nix

```diff
@@ -2,11 +2,11 @@
 
 buildDunePackage rec {
   pname = "mirage-unix";
-  version = "4.0.1";
+  version = "5.0.0";
```

Comment:
```suggestion
  version = "5.0.1";
```
`5.0.1` is available. Is `5.0.0` still desirable?

Reply:
> minimalOCamlVersion = "4.06";
Is this necessary?

Reply:
What do you mean?

Reply:
`minimalOCamlVersion` is missing but as it is `4.06` wondered if `4.06` is old enough to be disregarded. (Which is the case with `4.02`.) I have no opinion on this. You know better.